app/javascript/src/components/SceneEditor.tsx

```diff
@@ -44,37 +40,46 @@ class SceneEditor extends React.Component<SceneEditorProps> {
           />
         </div>
 
-        <div className="SceneEditorField">
-          <label className="SceneEditorHeading" htmlFor="content">Content</label>
-          <textarea name="content" ref={this.editor} defaultValue={text} />
-        </div>
+        <SceneEditorTextAreaField
+          name="content"
+          title="Content"
+          defaultValue={text}
+          placeholderText=""
+          instructionalText=""
+          onChange={this.onChangeContent.bind(this)}
```

Comment:
The style in this app so far is not to bind in render, but to define these functions a class propertires so onChangeContent should be defined like:

```
onChangeContent = (html: string) => {
// code here
}
```

Reply:
Addressed in 97241e4